.travis.yml

```diff
@@ -1,11 +1,28 @@
 sudo: required
-services:
-  - docker
 language: python
 python: "3.6"
+
+env:
+    - MONGODB_VER=mongodb-linux-x86_64-2.6.12 WIRED_TIGER=false
+    - MONGODB_VER=mongodb-linux-x86_64-3.4.16 WIRED_TIGER=false
+    - MONGODB_VER=mongodb-linux-x86_64-3.4.16 WIRED_TIGER=true
+    - MONGODB_VER=mongodb-linux-x86_64-3.6.6  WIRED_TIGER=false
+    - MONGODB_VER=mongodb-linux-x86_64-3.6.6  WIRED_TIGER=true
```

Comment:
Why repeat the same env var 5 times? Maybe add a comment about why you do this in here.

Reply:
Done

Reply:
:+1: